### PR TITLE
Fix issue in approval request reducer

### LINKED ISSUE #286: Approval requests are missing user info
"Requested by" and "Approved by" are blank.

image

### PR DESCRIPTION
**Pull request by rehandalal**
Fixes #286.

r?

### CODE REVIEW

File: src/tests/state/approvalRequests/reducers.test.js [hunk 1]
```diff
@@ -55,20 +56,23 @@ describe('Approval requests reducer', () => {
   });
 
   it('should handle RECIPE_HISTORY_RECEIVE', () => {
-    const appRequest = {
-      id: 'test',
-      arbitrary_data: 123,
+    const reducedApprovalRequest = {
+      ...approvalRequest,
+      approver_id: approvalRequest.approver ? approvalRequest.approver.id : null,
```
Comment (magopian) [inline src/tests/state/approvalRequests/reducers.test.js]: Why this test? Don't we know at this point what the test data is?

Comment (rehandalal) [reply src/tests/state/approvalRequests/reducers.test.js]: I'm not sure I copied it over from above, but I'll do some digging and remove if unnecessary.

**Comment by rehandalal** [comment]
bors=@magopian

**Comment by rehandalal** [comment]
merge conflict fixed. let's try this again...

bors r=@magopian

**Pull request merged**